src/server/services/cla.js

```diff
@@ -271,13 +271,18 @@ module.exports = function () {
                     query.repoId = repo.repoId;
                     findCla();
                 });
+            } else if (args.repoId) {
+                query.repoId = args.repoId;
+                findCla();
             } else if (args.orgId) {
                 query.ownerId = args.orgId;
                 query.org_cla = true;
                 findCla();
             } else {
-                query.repoId = args.repoId;
-                findCla();
+                deferred.reject();
+                if (typeof done === 'function') {
+                    done(new Error('Argument is not enough.'), null);
```

Comment:
Concerns:
Can I change the implementation of get() function? I think findCLA() should first try to search repo cla and then search org cla instead of based on args.orgId or args.repoId.